Blaise-Shyaka/Politico
# [#171592338] Create page allowing a user to run for an office

# What does this PR do?

It creates a page allowing users/a politician to run for  a government office they like

# Description of tasks to be completed

- Create a page where a user can choose which office to run for
- Make the page responsive

# How should this be manually tested?

- Clone this repository
- In `/templates/html/` open `become-candidate.html`
- Check the presence of a form to fill in the type of office, office name, and affiliated political party
- Resize browser to verify its responsiveness

# What are the relevant pivotal tracker stories?

#171592338

# Screenshots

![Screenshot (101)](https://user-images.githubusercontent.com/52489421/76042470-8408fc00-5f5d-11ea-8336-e46c9ae1c942.png)
![Screenshot (103)](https://user-images.githubusercontent.com/52489421/76042505-97b46280-5f5d-11ea-9f1f-2692bd572056.png)
![Screenshot (102)](https://user-images.githubusercontent.com/52489421/76042515-a13dca80-5f5d-11ea-92e8-fed269e4123b.png)
